src/media/js/site/login.js

```diff
@@ -4,9 +4,10 @@ import {connect} from 'react-redux';
 import * as tosActions from './actions/tos';
 import {Page, PageSection} from './components/page';
 import TOSSignatureContainer from './containers/tos';
+import LoginHandler from './containers/login'
```

Comment:
hm, this adds some coupling to this library and to the application. maybe have a util on the app.js level to pass LoginHandler.